_layouts/default.html

```diff
@@ -78,5 +78,10 @@
 
 	</div>
 
+	{% if page.hasCharts %}
+		<script src="https://cdn.jsdelivr.net/npm/chart.js@2.9.3/dist/Chart.min.js" defer></script>
```

Comment:
As this is from a third party domain and is a third-party script, we should use sub-resource integrity attributes to protect ourselves in case this script start's serving different JavaScript than what we want it to serve.
```suggestion
		<script src="https://cdn.jsdelivr.net/npm/chart.js@2.9.3/dist/Chart.min.js" integrity="sha256-R4pqcOYV8lt7snxMQO/HSbVCFRPMdrhAFMH+vr9giYI=" crossorigin="anonymous" defer></script>
```